src/components/right_page/Gallery.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import deptEvent from "../../assets/img/dept_event.png";
+import deptMkt from "../../assets/img/dept_mkt.png";
+import deptPr from "../../assets/img/dept_pr.png";
+import deptTech from "../../assets/img/dept_tech.png";
+
+//padding and width for image should be adjusted when replacing image source
+export default function Gallery(props){
+    return(
+        <div className="gallery-container">
+            <div className="row">
+                <div className="col-12 col-md-6 col-lg-4">
+                    <div className="text-us">
+                        <h1 style={{"font-size": "3.5rem"}}>我們</h1>
```

Comment:
![image](https://user-images.githubusercontent.com/39153483/105641766-d7c15400-5e53-11eb-9f06-4fef876efdf2.png)
This h1 element is blocking me from clicking the left button. I think you can push the h1 element to the behind by using the 'z-index' attribute in css/scss